### PR TITLE
[No-30][training]feedback-manager-user

### PR DESCRIPTION
**Pull request by phantuan1803**
- Fix feedback code manager user from pull request https://github.com/loitq/training/pull/30
- Checked : phpcs.phar
- Link task: https://docs.google.com/spreadsheets/d/1aWRDQ4TsW6BgTjm7iLPnI7R9VRPeZF0Fl6lEq-wfpa8/edit?folder=1KnNubuBhStnVllQ-62F3SW1mG3x9DT-G#gid=0&range=A49

### CODE REVIEW

**Comment by nguyentienthinh** [comment]
LGTM

**Review by nguyentienthinh — APPROVED** [review]

File: app/Http/Controllers/Admin/AdminController.php [hunk 1]
```diff
@@ -157,7 +165,13 @@ public function handleUserEdit(Request $request, $id)
     public function handleDelete($id)
     {
         $user = User::find($id);
+        if (!isset($user)) {
+            return redirect('/admin/user/list')
+                ->with('message', 'Could not be deleted');
+        }
+
         $user->delete();
```
Comment (ghost) [inline app/Http/Controllers/Admin/AdminController.php]: Suggest check by try... cath().
try{
       $user-> delete();
    }
    catch(\Exception $e){
       // do task when error
       return messageError = $e
    }

Comment (phantuan1803) [reply app/Http/Controllers/Admin/AdminController.php]: Fixed

**Comment by ghost** [comment]
LGTM

**Review by hienl — APPROVED** [review]
LGTM

**Pull request merged**